.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -184,7 +184,7 @@ jobs:
       # https://github.com/KDAB/cxx-qt/issues/121
       #
       # automake is needed for building libicu which is a dependency of Qt
-      run: brew install automake autoconf-archive ninja clang-format
```

Comment:
In the end, moving to macOS 12 has made installing clang-format fast again ? So do we need to worry about this ? As we'll still want `cargo test` on all platforms i think ?